### PR TITLE
EVG-19303 Revert Konami Code modal but keep easter egg

### PR DESCRIPTION
**Pull request by khelif96**
EVG-19303

### Description
Reverts the modal showing the konami code but lets keep the easter egg around cc @ablack12

### CODE REVIEW

**Review by ablack12 — APPROVED** [review]
Didn't read code but approving the easter egg keeping 😄

File: src/hooks/useKonamiCode/useKonamiCode.test.tsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    );
+    expect(
+      cache.extract()[
+        cache.identify({
+          __typename: "Task",
+          id: taskQuery.task.id,
+          execution: taskQuery.task.execution,
+        })
+      ].status
+    ).toBe("success");
```
Comment (minnakt) [inline src/hooks/useKonamiCode/useKonamiCode.test.tsx:77]: oh this is funny, I didn't even realize that it did that LOLL

File: src/hooks/useKonamiCode/useKonamiCode.test.tsx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    jest.spyOn(global, "Audio").mockImplementation(
+      () =>
+        ({
+          play: audioPlayMock,
+        } as any)
+    );
+    window.HTMLMediaElement.prototype.play = audioPlayMock;
+
+    // Duplicate the cache so that the cache is not mutated
+    // const gqlCache = new InMemoryCache().restore(cache.extract());
```
Comment (minnakt) [inline src/hooks/useKonamiCode/useKonamiCode.test.tsx:39]: (appears to be unused)

```suggestion
```

File: src/hooks/useKonamiCode/useKonamiCode.test.tsx [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    // eslint-disable-next-line testing-library/no-unnecessary-act
+    act(() => {
+      userEvent.type(
+        document.body,
+        "ArrowUpArrowUpArrowDownArrowDownArrowLeftArrowRightArrowLeftArrowRightba"
+      );
+    });
+
+    await waitFor(() => {
+      expect(audioPlayMock).toHaveBeenCalledWith();
```
Comment (minnakt) [inline src/hooks/useKonamiCode/useKonamiCode.test.tsx:62]: nit. (just preference)
```suggestion
      expect(audioPlayMock).toHaveBeenCalledTimes(1);
```

File: src/hooks/useKonamiCode/index.ts [hunk 4]
```diff
@@ -8,13 +9,20 @@ import { TaskStatus } from "types/task";
...
 
-const useKonamiCode = (onActivate: () => void) => {
+const useKonamiCode = () => {
   const [pressedKeys, setPressedKeys] = useState<string[]>([]);
   const dispatchToast = useToastContext();
   const { sendEvent } = useAprilFoolsAnalytics();
-  const downHandler = ({ key }: KeyboardEvent) => {
+  const downHandler = ({ key, target }: KeyboardEvent) => {
+    // Ignore key presses if the user is typing in an input
+    if (target instanceof HTMLInputElement) return;
```
Comment (minnakt) [inline src/hooks/useKonamiCode/index.ts:18]: If you want to cover more bases you could add `HTMLTextAreaElement`, `HTMLButtonElement`, and `HTMLSelectElement`

**Review by minnakt — APPROVED** [review]
🥳

**Comment by khelif96** [comment]
evergreen merge

**Pull request merged**